src/convert.cpp

```diff
@@ -225,3 +225,17 @@ Napi::Value from_ddwaf_object(ddwaf_object *object, Napi::Env env, int depth) {
 
   return result;
 }
+
+Napi::Object from_ddwaf_ruleset_info(ddwaf_ruleset_info ruleset_info, Napi::Env env) {
```

Comment:
wouldn't we want a pointer here for ruleset_info ?
that would allow us to put `ddwaf_ruleset_info_free` directly in this function, wdyt ?

Reply:
I wanted to keep the responsibility of freeing `ddwaf_ruleset_info` out of the convert function, leaving it with one single responsibility.

Reply:
Ok I'm fine with that :+1: